### PR TITLE
fix([sc-1495]): input error icon alignment

### PR DESCRIPTION
**Pull request by chorniy8pavlo**

### CODE REVIEW

File: packages/nfid-sdk-react/src/atoms/input/index.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
           />
 
           {errorText && (
-            <span className="absolute right-3 top-[14px]">
+            <span className='absolute right-3 top-[50%] translate-y-[-50%]'>
```
Comment (plitzenberger) [inline packages/nfid-sdk-react/src/atoms/input/index.tsx:64]:
```suggestion
            <span className='absolute right-3 top-1/2 translate-y-1/2'>
```

File: packages/nfid-sdk-react/src/atoms/input/index.tsx [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-  placeholder?: string;
-  type?: string;
-  icon?: ReactElement;
-  errorText?: string;
-  helperText?: string;
-  labelText?: string;
-  pin?: boolean;
-  small?: boolean;
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
```
Comment (plitzenberger) [inline packages/nfid-sdk-react/src/atoms/input/index.tsx:7]:
```suggestion
  extends React.InputHTMLAttributes<HTMLDivElement> {
```

**Review by plitzenberger — CHANGES_REQUESTED** [review]
Please prevent arbitrary styles when possible

File: packages/nfid-sdk-react/src/atoms/input/index.tsx (hunk 2 above)
Comment (chorniy8pavlo) [reply packages/nfid-sdk-react/src/atoms/input/index.tsx:7]: why HTMLDiv? It's Input component

**Review by QuopHi-Geek — APPROVED** [review]

**Pull request merged**